extensions/src/main/java/org/pentaho/platform/plugin/action/mondrian/catalog/MondrianCatalogHelper.java

```diff
@@ -345,6 +351,24 @@ public static int addToCatalog( String baseUrl, boolean enableXmla, String schem
   }
 
 
+  /**
+   * Add a catalog to the Mondrian Catalog Service.
+   *
+   * @param baseUrl            the base URL for the XMLA service, e.g. http://localhost:8080/pentaho/
+   * @param enableXmla         Deprecated, use the baseUrl to determine if XMLA is enabled
+   * @param schemaSolutionPath the solution path to the schema definition, e.g. /etc/mondrian/XXX/schema.xml
+   * @param session            Pentaho session
+   * @param jndiName           the JNDI name of the datasource to use for this catalog, e.g. java:comp/env/jdbc/XXX
+   * @param overwrite          flag to indicate if the catalog should be overwritten if it already exists
+   * @return 0 is success, -1 if an error occurred
+   * @deprecated Use {@link #addToCatalog(String, IPentahoSession, String, boolean)} instead.
+   */
+  @Deprecated( forRemoval = true )
```

Comment:
Can't we take the opportunity to delete this deprecated method?

Reply:
I had some doubts regarding the removal of public methods because of code references.